observability/src/main/java/io/grpc/observability/Observability.java

```diff
@@ -42,14 +45,13 @@ public static synchronized Observability grpcInit() throws IOException {
     if (instance == null) {
       GlobalLoggingTags globalLoggingTags = new GlobalLoggingTags();
       ObservabilityConfigImpl observabilityConfig = ObservabilityConfigImpl.getInstance();
-      Sink sink = new GcpLogSink(observabilityConfig.getDestinationProjectId());
+      Sink sink = new GcpLogSink(observabilityConfig.getDestinationProjectId(),
+          globalLoggingTags.getLocationTags(), globalLoggingTags.getCustomTags(), 10);
```

Comment:
I am a bit concerned that value of 10 for flushLimit is too low and could impact RPC performance. I think the Cloud Logging library has the limit as 1000 for a reason. Something like 100 or 200 looks like a good value. In any case we can tweak it later.